### ISSUE TITLE
WKWebViewJavascriptBridge should not directly call "stopLoading" of a WKWebView

### ISSUE THREAD

**Issue by raxcat** [marcuswestin/WebViewJavascriptBridge#176]
Hi there, love this project.

I am facing an issue regarding to WKWebViewJavascriptBridge. 

https://github.com/marcuswestin/WebViewJavascriptBridge/blob/master/WebViewJavascriptBridge/WKWebViewJavascriptBridge.m  
Line 130 made a function call:
```
 [webView stopLoading];
```
I would suggest the following instead: (Use decisionHanlder to deny/cancel requests whose scheme are "wvjbscheme")
```
decisionHandler(WKNavigationActionPolicyCancel);
```
I have compared to the WebViewJavascriptBridge. In the WebViewJavascriptBridge, UIWebViewDelegate method 
``` 
- (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)request navigationType:(UIWebViewNavigationType)navigationType
```
returns NO instead of stopping the whole UIWebView.

Stop the loading of a WKWebView directly stop all pending request. Especially sites under HTTPS. Dunno why but thats what I saw.

**Comment by yirenjun** [comment]
nice

**Comment by marcuswestin** [comment]
This has been fixed in v5.0.1.

When you switch to the new version, please note that the API has changed. In particular, make sure that you use the javascript setup code, as it has changed: https://github.com/marcuswestin/WebViewJavascriptBridge#usage

Cheers!
